e2e/test_get_code.py

```diff
@@ -38,38 +59,62 @@ def test_limited_is_contract(self):
             self.expected
         )
 
+    @drop_files
     def test_results(self):
-        contract_detector = ContractDetector(
+        contract_detector = EvmAccountInfo(
             max_batch_size=2,
             node_url=NODE_URL,
-            addresses=self.addresses
+            addresses=self.addresses,
+            network='mainnet'
         )
 
-        self.assertEqual(contract_detector.results(), self.expected)
+        self.assertEqual(contract_detector.contracts(self.dummy_file), self.expected)
 
     def test_fail_with_bad_input(self):
-        contract_detector = ContractDetector(
+        contract_detector = EvmAccountInfo(
             max_batch_size=2,
             node_url=NODE_URL,
-            addresses=["Bad Input"]
+            addresses=["Bad Input"],
+            network='mainnet'
         )
         with self.assertRaises(IOError):
-            contract_detector.results()
+            contract_detector.contracts(self.dummy_file)
 
+    @drop_files
     def test_limitations(self):
         num_items = 1000
         long_list = [
             "0x" + f"{i}".zfill(40)
             for i in range(num_items)
         ]
-        detector = ContractDetector(
-            max_batch_size=num_items // 10,
+        detector = EvmAccountInfo(
+            max_batch_size=num_items // 3,
```

Comment:
I wanted to put less stress on the network by making fewer over all request (this speeding up the run time of this test - in case we move it out of e2e and into the actual test suite that gets run in CI.